src/input_utils.py

```diff
@@ -172,7 +172,7 @@ def _decode_fn(record):
 
     # Image
     # We follow the implementation of ViT
-    im = tf.io.decode_image(example['image_data'], dtype=tf.float32)
+    im = tf.io.decode_image(example.pop('image_data'), dtype=tf.float32)
```

Comment:
Just curious why making this change? Is it not working for "example['image_data']"?

Reply:
Since we will not use this while training so I pop it out. Will it slow down training if we pop it out?

Reply:
I guess it is fine.